Aerolyzer/app/templates/app/faq.html

```diff
@@ -11,7 +11,7 @@
 			</div>
 		</div>
 	</div>
-	

```

Comment:
on line 5, " is missing, should be 
<div class="container" style="height: 375px;">